modules/data.land/R/match_species_id.R

```diff
@@ -0,0 +1,65 @@
+#' Match BETY species ID.
+#'
+#' Parses species codes in input data and matches them with the BETY species ID.
+#' 
+#' @param input_codes Character vector of species codes
+#' @param format_name Species code format name (character, length 1)
+#' @param betydb \code{dplyr} \code{src} object containing BETY connection 
+#' @param custom_translation_path File path to translation
```

Comment:
I thought we were going to overload this arg to be either the path or the table itself. In which case you might to rename to translation_table